core/controllers/feedback.py

```diff
@@ -215,5 +220,33 @@ def get(self, exploration_id):
 class UnsentFeedbackEmailHandler(base.BaseHandler):
     """Handler task of sending emails of feedback messages.
     This is yet to be implemented."""
+
     def post(self):
-        pass
+        payload = json.loads(self.request.body)
+        user_id = payload['user_id']
+        references = feedback_services.get_feedback_message_references(user_id)
+        transaction_services.run_in_transaction(
+            feedback_services.update_feedback_email_retries, user_id)
+
+        messages = {}
+        for reference in references:
+            message = feedback_services.get_message(
+                reference.exploration_id, reference.thread_id,
+                reference.message_id)
+
+            exploration = exp_services.get_exploration_by_id(
+                reference.exploration_id)
+
+            message_text = message.text
+            if len(message_text) > 200:
+                message_text = message_text[:200] + '...'
+
+            if exploration.title in messages:
```

Comment:
Here we are talking about one user's exploration, chances of having 2 same named exploration for a single user seems rare to me. In any case when messages are sent to the user in email it will have structure as:

> Title: Message1
> Title: Message2

So even if above 2 explorations are different (but having same title) there is no way to find difference. So it is all same on outside, using exploration id as key would not make any difference. Do you still want me to do it ?